### PR TITLE
Add_test_identifier_for_isRedhatRelease

### PR DESCRIPTION
**Pull request by shimritproj**

### CODE REVIEW

**Comment by HamadiSE** [comment]
@shimritproj can you please solve the conflicts ?

File: test-network-function/identifiers/identifiers.go [hunk 1]
```diff
@@ -489,4 +494,15 @@ the changes for you.`,
 			scale-in oc command for (N-1) replicas. Lastly, it executes the scale-out oc command, restoring the original replicaCount of the deployment/s.`),
 		Remediation: `Make sure CNF deployments/replica sets can scale in/out successfully.`,
 	},
+	TestIsRedHatReleaseIdentifier: {
+		Identifier: TestIsRedHatReleaseIdentifier,
+		Type:       normativeResult,
+		Description: formDescription(TestIsRedHatReleaseIdentifier,
```
Comment (HamadiSE) [inline test-network-function/identifiers/identifiers.go:500]: this description does not match the test. Needs to be updated.

Comment (HamadiSE) [reply test-network-function/identifiers/identifiers.go:500]: this text is meant for the end user to figure out what's wrong with the test case and some hints on how to fix it.

**Review by HamadiSE — CHANGES_REQUESTED** [review]

File: test-network-function/platform/suite.go [hunk 2]
```diff
@@ -78,9 +79,36 @@ var _ = ginkgo.Describe(common.PlatformAlterationTestKey, func() {
...
 })
 
+// testIsRedHatRelease fetch the configuration and test containers attached to oc is Red Hat based.
+func testIsRedHatRelease(configData *common.ConfigurationData) {
+	ginkgo.It("Should report a proper Red Hat version", func() {
+		defer results.RecordResult(identifiers.TestIsRedHatReleaseIdentifier)
+		for _, cut := range configData.ContainersUnderTest {
+			testContainerIsRedHatRelease(cut)
+		}
+		testContainerIsRedHatRelease(configData.TestOrchestrator)
```
Comment (jun-rh) [inline test-network-function/platform/suite.go:94]: Remove this line. No need to test our own container.

File: test-network-function/identifiers/identifiers.go [hunk 3]
```diff
@@ -489,4 +494,11 @@ the changes for you.`,
...
+			`This test is meant for the end user to figure out what's wrong with the test case and some hints on how to fix it.`),
+		Remediation: `You should recreate the nodes, recreating is recommended because there might be other unknown changes`,
```
Comment (HamadiSE) [inline test-network-function/identifiers/identifiers.go:502]: my comments were explanation and example, please update the description to indicate what the test is doing actually.

**Review by HamadiSE — CHANGES_REQUESTED** [review]

File: test-network-function/platform/suite.go [hunk 4]
```diff
@@ -78,9 +79,35 @@ var _ = ginkgo.Describe(common.PlatformAlterationTestKey, func() {
 			testSysctlConfigs(env)
 		}
 
+		testIsRedHatRelease(env)
```
Comment (HamadiSE) [inline test-network-function/platform/suite.go:82]: this should be inside if block.
it should in line 80.

**Review by HamadiSE — APPROVED** [review]

**Review by wying3 — APPROVED** [review]

**Pull request merged**